### PR TITLE
add: auth

### PR DESCRIPTION
**Pull request by thegeorgenikhil**
Added
- signin
- signup
- signout
- loaders

### CODE REVIEW

**Review by ravikumar1002 — APPROVED** [review]
looks good to merge

File: src/pages/Signin/Signin.jsx [hunk 1]
```diff
@@ -1,44 +1,107 @@
...
-            className="input"
-            placeholder="Enter Password: "
-            name="password"
-          />
-        </div>
-        <div className="flex justify-between text-xs align-center">
-          <div className="flex align-center">
-            <input type="checkbox" name="remember-me" />
-            <label htmlFor="remember-me">&#160;Remember Me</label>
+        <form action="">
```
Comment (manish591) [inline src/pages/Signin/Signin.jsx]: Remove `action=""`

File: src/pages/Signin/Signin.jsx [hunk 2]
```diff
@@ -1,44 +1,107 @@
...
+          <div className="input-group">
+            <label htmlFor="email">Email</label>
+            <input
+              type="text"
+              id="email"
+              className="input"
+              placeholder="Enter Email:"
+              name="email"
+              onChange={formChangeHandler}
+              value={email}
```
Comment (manish591) [inline src/pages/Signin/Signin.jsx]: I think you should use the `required` attribute here to let users know that these values need to fill.

File: src/App.js [hunk 3]
```diff
@@ -10,21 +10,24 @@ import {
   Signup,
 } from "./pages";
 import { Navbar } from "./components";
+import { AuthProvider } from "./context";
 function App() {
```
Comment (manish591) [inline src/App.js]: Use the arrow function here also as you have used arrow function in  your another components

**Review by manish591 — APPROVED** [review]

**Pull request merged**